Content.Server/Physics/Controllers/MoverController.cs

```diff
@@ -134,37 +133,43 @@ protected override void HandleFootsteps(IMoverComponent mover, IMobMoverComponen
 
             mobMover.StepSoundDistance -= distanceNeeded;
 
-            if (mover.Owner.TryGetComponent<InventoryComponent>(out var inventory)
-                && inventory.TryGetSlotItem<ItemComponent>(EquipmentSlotDefines.Slots.SHOES, out var item)
-                && item.Owner.TryGetComponent<FootstepModifierComponent>(out var modifier))
-            {
-                modifier.PlayFootstep();
```

Comment:
had to modify this clause and move it down, since we are no longer using the old approach here